sql/plan.go

```diff
@@ -80,6 +80,8 @@ func (p *planner) makePlan(stmt parser.Statement) (planNode, error) {
 		return p.Select(n)
 	case *parser.Set:
 		return p.Set(n)
+	case *parser.SetTransaction:
+		return p.SetTransaction(n)
```

Comment:
I'm not clear on what you're suggesting. `client/txn` will return an error if the transaction is already running. `sql.Executor.execStmt()` will return an error if no transaction is in progress. Am I missing an error condition?